src/go/rpk/pkg/cli/acl/acl.go

```diff
@@ -83,6 +83,21 @@ in the --allow-principal and --deny-principal flags. This command will add the
 "User:" prefix for you if it is missing. The wildcard '*' matches any user.
 Creating an ACL with user '*' grants or denies the permission for all users.
 
+ROLES
+
+Alternatively, an ACL may be bound to a role. To create a role-bound ACL,
+follow the same procedure used to create a user ACL, but use the --allow-role
```

Comment:
does this applies only to `rpk acl create user` ? 

Why do we explicitly have the following? Can't we just explain the process here? (even if it contains duplication)

> follow the same procedure used to create a user ACL

Reply:
Can we have one example showing how to create the role?

Reply:
> does this applies only to rpk acl user create ?
No this is `rpk acl create`

Reply:
>even if it contains duplication

fine by me! my only concern would be keeping the two in sync, but it's unlikely to change. anything is better than trying to thread the role concept back through existing text IMHO.